### PR TITLE
Milestone 2: Feature/api manipulation

### PR DESCRIPTION
**Pull request by alvp01**
# In this milestone:

- Create a new game with the name of your choice by using the API.
- Implement the "Refresh" button (receiving data from the API and parsing the JSON).
- Implement the form "Submit" button (sending data to the API).
- Use async and await JavaScript features to consume the API.
- No styling is required.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1 +1,21 @@
...
+    displayScores(response.result);
+  });
+});
+
+refreshBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  getScores().then((response) => {
+    displayScores(response.result);
+  });
+});
```
Comment (Lameck1) [inline src/index.js]: - [x] [OPTIONAL] Add one more EventListener to have the list populated when the window is loaded. This is just for a good user experience. For example 👇🏼

```JavaScript
window.addEventListener('load', (event) => {
  // do something here with this event listener
});
```

File: src/index.js [hunk 2]
```diff
@@ -1 +1,21 @@
...
+import './style.css';
+import { getScores, addScore } from './modules/scoresHandle.js';
+import { displayScores } from './modules/createList.js';
+
+const submitBtn = document.querySelector('.btn-submit');
+const refreshBtn = document.querySelector('.btn-refresh');
+
+submitBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  addScore();
```
Comment (Lameck1) [inline src/index.js:11]: - [x] [OPTIONAL] For a good user experience, you may consider emptying the input fields when a score is submitted.

File: src/index.js [hunk 3]
```diff
@@ -1 +1,21 @@
...
+  getScores().then((response) => {
+    displayScores(response.result);
+  });
+});
+
+refreshBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  getScores().then((response) => {
+    displayScores(response.result);
+  });
```
Comment (Lameck1) [inline src/index.js]: - [ ] [OPTIONAL] We are encouraged to use Object Destructuring in ES6. This block of code can also be written as:

```JavaScript
refreshBtn.addEventListener('click', (e) => {
  e.preventDefault();
  getScores().then(({ result }) => {
    displayScores(result);
  });
});
```
Notice this line where the Destructuring has taken place 👉🏼 ```getScores().then(({ result }) => {```

You can consider refactoring this.

**Review by Lameck1 — APPROVED** [review]
Hi, @alvp01

Your project is complete!  It's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* Professional README ✔️ 
* Good commit messages ✔️
* Cleaner codebase ✔️
* Required Linters Set up ✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by alvp01** [comment]
> Hi, @alvp01
> 
> Your project is complete! It's time to merge it :shipit:
> Congratulations! tada
> ## Highlights
> 
>     * Professional README heavy_check_mark
> 
>     * Good commit messages heavy_check_mark
> 
>     * Cleaner codebase heavy_check_mark
> 
>     * Required Linters Set up heavy_check_mark
> 
> 
> Cheers and Happy coding!clapclapclap
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Thank you for your time and work! Happy weekend.

**Pull request merged**